docs/dev/api/insights.md

```diff
@@ -1186,26 +1388,249 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 
 ```jsx title="Sample Response"
 {
-  "additionalProp1": {
-    "ccy_exec_mean": [
-      0
-    ],
-    "ccy_exec_peak": [
-      0
-    ],
-    "datestamp": [
-      "2023-04-18"
-    ],
-    "jobs": [
-      0
-    ],
-    "minutes": [
-      0
-    ]
-  },
-  "additionalProp2": {
-    "ccy_exec_mean": [
-      0
+  "automation_backend": [
+    {
+      "name": "<automation_backend>",
+      "count": 5
+    }
+  ],
+  "browser": [
+    {
+      "name": "<browser>",
+      "count": 7
+    }
+  ],
+  "build": [
+    {
+      "name": "<build>",
+      "count": 2
+    }
+  ],
+  "os": [
+    {
+      "name": "<os>",
+      "count": 4
+    }
+  ],
+  "device": [
+    {
+      "name": "<device>",
+      "count": 8
+    }
+  ],
+  "tag": [
+    {
+      "name": "<tag>",
+      "count": 8
+    }
+  ]
+}
+```
+
+</details>
+
+## Activity
+
+### Get Activity
+
+<details><summary><span className="api get">GET</span> <code>/rest/v1/users/&#123;user_id&#125;/activity</code></summary>
+<p/>
+
+Return daily statistics about test results and concurrency usage for the specified <code>user_id</code>.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>user_id</code></td>
+       <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Return only jobs that belongs to the specified <code>user_id</code></p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>since</code></td>
+       <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return only jobs ran from the provided Unix timestamp on.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>until</code></td>
+       <td><p><small>| QUERY | OPTIONAL | STRING|</small></p><p>Return only jobs ran until the provided Unix timestamp.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>level</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>
+     Specifies the ownership level. Supported values are: <code>user</code>, <code>organization</code>
```

Comment:
When there is more than one available value, we should use a bulleted list so the description is easier to read.
```suggestion
     Specifies the ownership level. Supported values are: <ul><li><code>user</code></li></li><code>organization</code></li></ul>
```